deploy/pks/1-collector-cluster-role.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  annotations:
+    rbac.authorization.kubernetes.io/autoupdate: "true"
+  creationTimestamp: null
+  labels:
+    kubernetes.io/bootstrapping: rbac-defaults
+  name: wavefront-collector
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  - namespaces
+  - nodes
+  - nodes/stats
+  - pods
+  - services
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - extensions
+  resources:
+  - deployments
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
Can you add these lines as well: https://github.com/wavefrontHQ/wavefront-kubernetes-collector/blob/d66cbe6b6bfa81f2887d4e9b8fe532bf9569d555/deploy/kubernetes/1-collector-cluster-role.yaml#L32